.travis.d/definition_check.py

```diff
@@ -80,9 +82,10 @@ def read_nids():
                         nids = user_nids
                 if SECTION != 'functions':
                     continue
-                if nids.get(k):
+                if current_nids.get(k):
```

Comment:
imo both are needed.
```
if nids.get(k):
  errors.append(...)
elif current_nids.get(k):
  errors.append(...)
```
but, actually i want to remove ambiguous conflicting case between libs.
after the merging this PR, we can't load ScePaf and SceLibc at sametime
also compiler requires nostdlib.

so my opinion is just change the function name of the ScePaf.
for example; `memcpy/memset` to `sce_paf_memcpy/memset`(or `sce_paf_libc_memcpy/memset`) or just don't add them to above conflicting with `0x2C5B6F9C/0xE148AF94`
imo we only need one of them. isn't it?

Reply:
Yes. I have never seen such a case at SCE.
But, if project requires ScePaf and want to include external library sources, all memcpy/memset must be rewritten to sce_paf_memcpy/sce_paf_memset.

But now it doesn't matter. So memcpy/memset will be commented out for the future.